### ISSUE TITLE
v3.3.[4,5,6] has a sourceMappingURL comment in query-monitor.css but no source map is included in the zip

### ISSUE THREAD

**Issue by pbiron** [johnbillion/query-monitor#443]
[note: 3.3.3, and probably previous releases as well, had no sourceMappingURL comment]

This is causing a problem for me when debugging a theme using Chrome 74, because after the main WP request Chrome is making another that results in a 404...and this causes `is_404()` to incorrectly return `true` part way through the theme's rendering of the page.

**Comment by johnbillion** [comment]
Thanks I'll get that fixed up.

> and this causes `is_404()` to incorrectly return `true` part way through the theme's rendering of the page.

This doesn't sound right to me. A 404 in a client-side request can't change the state of a server-side conditional on the originating page.

**Comment by pbiron** [comment]
> > and this causes `is_404()` to incorrectly return `true` part way through the theme's rendering of the page.
> 
> This doesn't sound right to me. A 404 in a client-side request can't change the state of a server-side conditional on the originating page.

Of course, you are correct.  Last night I figured out what was going on and was going to come back and correct that statement.

I was setting a break point in one of the theme's templates (as well as another in part of core).  While stopped at that breakpoint the browser was issuing the request for the source map.  So, when I'd resume from the theme breakpoint I'd then stop at the core breakpoint...and I got confused :-)  It was a long day yesterday :-)

But, yeah, fixing the missing sourcemap would still be good.

thanx.
